doc/src/python.rst

```diff
@@ -80,85 +81,97 @@ Examples
 
    python source funcdef.py
 
-   python source inline "from lammps import lammps"
+   python source here "from lammps import lammps"
 
 
 Description
 """""""""""
 
-Define a Python function or execute a previously defined function or
-execute some arbitrary python code.  Arguments, including LAMMPS
-variables, can be passed to the function from the LAMMPS input script
-and a value returned by the Python function to a LAMMPS variable.  The
-Python code for the function can be included directly in the input
-script or in a separate Python file.  The function can be standard
-Python code or it can make "callbacks" to LAMMPS through its library
-interface to query or set internal values within LAMMPS.  This is a
-powerful mechanism for performing complex operations in a LAMMPS input
-script that are not possible with the simple input script and variable
-syntax which LAMMPS defines.  Thus your input script can operate more
-like a true programming language.
+The *python* command allows to interface LAMMPS with an embedded Python
+interpreter and enabled to either execute arbitrary python code in that
```

Comment:
and enables either executing arbitrary ... registering a Python function ...or invoking a previously registered function